list/harvesterhci.io.dashboard/index.vue

```diff
@@ -177,6 +179,27 @@ export default {
         });
 
         if (resource.spoofed) {
+          if (resource.spoofed?.filterNamespace && Array.isArray(resource.spoofed.filterNamespace)) {
+            const clusterCounts = this.$store.getters['harvester/all'](COUNT)[0].counts;
+            const statistics = clusterCounts[resource.type] || {};
+
+            for (let i = 0; i < resource.spoofed.filterNamespace.length; i++) {
+              const nsStatistics = statistics?.namespaces[resource.spoofed.filterNamespace[i]] || {};
+
+              if (nsStatistics.count) {
+                out[resource.type]['useful'] -= nsStatistics.count;
+              }
+              Object.entries(statistics?.namespaces[resource.spoofed.filterNamespace[i]] || {}).forEach((entry) => {
```

Comment:
```suggestion
              Object.entries(statistics?.namespaces[resource.spoofed.filterNamespace[i]?.states] || {}).forEach((entry) => {
```

![image](https://user-images.githubusercontent.com/18737885/148381533-917bcede-d480-4482-a34a-2eb6f7211be8.png)

Reply:
Done,  PTAL, thanks